.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: Close and mark stale issue
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ github.token }}
+          stale-issue-message: "Oops, seems like we needed more information for this issue, please comment with more details or this issue will be closed in 7 days."
+          close-issue-message: "This issue was closed because it is missing author input."
+          stale-issue-label: "stale"
+          any-of-labels: "need-author-input"
+          days-before-issue-stale: 6
+          days-before-issue-close: 7
+          enable-statistics: true
```

Comment:
IIUC, statistics depend on [enabling logs](https://github.com/actions/stale?tab=readme-ov-file#debugging) - is it worth it?

Reply:
probably not at this time, we can re-visit in the future